test/unit/test-ampdoc.js

```diff
@@ -119,21 +108,6 @@ describe('AmpDocService', () => {
           return;
         }
 
-        const newAmpDoc = service.installShadowDoc(
```

Comment:
Should rename the test description to something like 'should yield the closest shadow doc'.

Reply:
good idea. done.